content/docs/deployment/clients/_index.md

```diff
@@ -521,6 +524,21 @@ sudo ./$filename --config client.config.yaml service install;
 - Scripts: `Script we created in 5.`
 - Triggers: Up to you.
 
+{{% notice note "Mac settings may not show FDA is enabled" %}}
+
+In **Settings > Privacy & Security > Full Disk Access** `velociraptor` can still appear as disabled, even when it is correctly configured in the MDM Profile. To ensure the FDA has been enabled, follow the verification steps:
+
+- Run the following script via Velociraptor itself:
+```
+ access_result=$(sudo sqlite3 /Library/Application\ Support/com.apple.TCC/TCC.db 'select * from access' 2>/dev/null | awk -F '|' '{print $2}' ); access_result_len=$(echo $access_result | wc -c); 
```

Comment:
How about collecting this https://docs.velociraptor.app/artifact_references/pages/macos.system.tcc/

Reply:
I haven't tested this Artifact, so I am not sure what the output would be **without** the FDA. I can try to test it Monday/Tuesday when I get access to the test device, but feel free to edit this part if you already know the output! 

I created that code only when I was testing MDM deployment, and once validated on a few devices, I pushed the profile to all, as it was a good confirmation that the profile works.

Reply:
Updated the docs to use TCC instead of custom code.